### DISCUSSION TITLE
All my hygrometers have gone down

### DISCUSSION THREAD

**Discussion by CriticalO** [custom-components/ble_monitor#1403] [Q&A] [answered]
Can you check this please, I've lost all my hygrometers restored from backup and they are all back except Thermo plus. Something is wrong with this update. Had to redownload the other day after 10.11.0 then they all worked then upgraded to 10.11.1, still working upgraded to latest hacs ble monitor then not big worked, had to restore again, so gonna leave the 12.17.0 update and have to put up with no temperature of humidity from my kitchen. Fix please thanks. lol the batteries were gone in my Thermoplus so hopefully after a reboot I'll get my kitchen reading back, I'm still definitely not updating though I lost all my readings I have Govee, about 9 xiaomi mi flashed firmware sensors in various places, 2 switch bot and obviously the thermoplus so I think I'll wait anyway thanks for all your hard work appreciated

**Comment by Ernst79** [comment, marked as answer]
That is quite a big jump in version, from 10 to 12. There have been a lot of updates in between, so hard to tell what’s wrong. Anything in the logs?

**Reply by CriticalO** [reply]
Hi there thanks for the quick reply I'm gonna leave it for now, can't live without knowing the temps everywhere but thanks anyway I'll wait and see what happens next time. 👍

Oh just to confirm ble monitor is always up to date (usually), I had 10.15.0 and the one that messed everything up was 10.17.0.

Much appreciated 😃

**Reply by Ernst79** [reply]
Don’t you mean 12.15.0 to 12.17.0??? 10.15.0 is a long time ago, whereas 12.15.0 is quite recent. That’s why I’m confused.

**Reply by CriticalO** [reply]
lol yes that's the one, I've tried so many backups to fix these hygrometers I don't know if I'm coming or going, ha itself is also always on the latest version 2024.11.1 or whatever the latest one is now so not sure what happened although I've lost a few updates to Zigbee automations that I changed today so I'll wait until I've remembered all the updates I did back everything up and try again with the next update. Thanks for you help.

**Reply by Ernst79** [reply]
Also check the batteries, empty batteries can also cause sensors to stop working.

 Or a full reboot of the HA machine, not only rebooting HA, but restart the entire device (power off, power on). Something’s the Bluetooth dongle gets stuck after an HA update, which can only be solved with a full reboot

**Reply by CriticalO** [reply]
Ok thanks yes as mentioned the Thermoplus batteries were gone although all the rest are fine and I've rebooted so many times now cause or restoring all these backups and adding new integrations so thanks you can close this now. I don't want to take the chance again at least until I've fixed my button automations, have a good one and thanks 😊
